scoring_board.h

```diff
@@ -0,0 +1,50 @@
+#ifndef SCORING_BOARD_H
+#define SCORING_BOARD_H
+
+#include <string>
+#include <iosfwd>
+
+class scoring_board
+{
+public:
+    scoring_board(unsigned int score_player1 = 0,
+                  unsigned int score_player2 = 0,
+                  unsigned int score_player3 = 0,
+                  unsigned int timer = 0,
+                  short winner = -1);
+    void set_score_player1(unsigned int score) {m_score_player1 = score;};
```

Comment:
```suggestion
    void set_score_player1(int score) {m_score_player1 = score;};
```
The value need to be negative for us to check that it is negative, `unsigend int` unfortunately can never be negative, so we will never catch the cultpript.